e2e_playwright/multipage_apps_v2/mpa_v2_basics_test.py

```diff
@@ -120,12 +120,12 @@ def test_supports_navigating_to_page_directly_via_url(app: Page, app_port: int):
     expect(page_heading(app)).to_contain_text("Page 5")
 
 
-def test_supports_navigating_to_page_directly_via_url_title(app: Page, app_port: int):
```

Comment:
ah probably not because of this change [here](https://github.com/streamlit/streamlit/pull/8778/files#diff-cde9e91039649c86b0e33690bf374e49cb91304e359cb681c2f4c4414e0d3859L140)